Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_001.HttpAuthorizationRequestHeader.sarif

```diff
@@ -10,267 +10,14 @@
           "product": "Microsoft.TestHost",
           "fullName": "testhost 15.0.0.0",
           "version": "15.0.0.0",
-          "semanticVersion": "15.0.0",
-          "rules": [
-            {
-              "id": "SEC101/001",
```

Comment:
@HulonJenkins Why do we need to remove this? Isn't it enough to just remove the rule from the plug-in?

Reply:
We leave this shell because we plan to re-enable the rule. Currently, the supporting rule test input is still being assessed (with no results because the rule is gone).